actions/run_action.go

```diff
@@ -82,6 +83,21 @@ func (run *RunAction) Verify(context *debos.DebosContext) error {
 		args := strings.Split(run.Script, " ")
 		run.scriptPath = debos.CleanPathAt(args[0], context.RecipeDir)
 		run.scriptArgs = args[1:]
+
+		/* Check the script exists on the filesystem (following symlinks) */
+		stat, err := os.Stat(run.scriptPath)
+		if err != nil {
+			return err
+		}
+
+		if stat.IsDir() {
+			return fmt.Errorf("script %s is a directory", run.Script)
+		}
+
+		/* Check the script is executable */
+		if stat.Mode()&0111 == 0 {
```

Comment:
executable and readable ideally; though i don't think golang gives you nice helpers for that unfortunately :/